iogt_content_migration/management/commands/load_v1_users.py

```diff
@@ -0,0 +1,387 @@
+import json
+from time import sleep
+
+import psycopg2
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management import BaseCommand
+from django.core.serializers.json import DjangoJSONEncoder
+from django_comments_xtd.models import XtdComment
+from pip._vendor.distlib.compat import raw_input
+from wagtail.contrib.forms.utils import get_field_clean_name
+
+from tqdm import tqdm
+
+from home.models import Article, SectionIndexPage
+from questionnaires.models import Survey, UserSubmission, SurveyIndexPage, Poll
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--host',
+            default='0.0.0.0',
+            help='IoGT V1 database host'
+        )
+        parser.add_argument(
+            '--port',
+            default='5432',
+            help='IoGT V1 database port'
+        )
+        parser.add_argument(
+            '--name',
+            default='postgres',
+            help='IoGT V1 database name'
+        )
+        parser.add_argument(
+            '--user',
+            default='postgres',
+            help='IoGT V1 database user'
+        )
+        parser.add_argument(
+            '--password',
+            default='',
+            help='IoGT V1 database password'
+        )
+
+        parser.add_argument(
+            '--skip-locales',
+            action='store_true',
+            help='Skip data of locales other than default language'
+        )
+
+        parser.add_argument(
+            '--delete-users',
+            action='store_true',
+            help='Delete existing Users and their associated data. Use carefully'
+        )
+
+    def handle(self, *args, **options):
+        self.db_connect(options)
+
+        self.registration_survey_mandatory_group_ids = self.request_registration_survey_mandatory_groups()
+
+        self.content_type_map = dict()
+        self.comments_map = dict()
+        self.articles_map = dict()
+        self.surveys_map = dict()
+        self.polls_map = dict()
+        self.users_map = dict()
+
+        self.delete_users = options.get('delete_users')
+
+        self.clear()
+        self.stdout.write('Existing site structure cleared')
```

Comment:
This is misleading as we're not clearing the site structure here. Remove this line.